go.mod

```diff
@@ -6,7 +6,7 @@ require (
 	github.com/nuclio/logger v0.0.1
 	github.com/nuclio/zap v0.0.2
 	github.com/pkg/errors v0.8.1
-	github.com/v3io/v3io-go v0.0.0-20190416000000-1c46bb377c604d66b977f179edc92949c0f0588d
+	github.com/v3io/v3io-go v0.0.0-20190418092154-5ef4514d19fbffd1b47468caf07f6dd9d1df1506
```

Comment:
is it still in development? when do you plane to release 0.0.1?

Reply:
You can discuss releases with @pavius :)